components/calendar/src/types.rs

```diff
@@ -355,9 +355,9 @@ impl MonthInfo {
         self.standard_code.parsed().map(|(_, l)| l).unwrap_or(false)
     }
 }
+
 /// A struct containing various details about the position of the day within a year. It is returned
-// by the [`day_of_year_info()`](trait.DateInput.html#tymethod.day_of_year_info) method of the
-// [`DateInput`] trait.
+/// by the `day_of_year_info()` method of the `DateInput` trait.
 #[derive(Copy, Clone, Debug, PartialEq)]
 #[allow(clippy::exhaustive_structs)] // this type is stable
```

Comment:
Why did you keep this one exhaustive?

Reply:
Hm, I thought I had fixed this already. Perhaps in the reshuffle of ctors it got undone